docs/00_install/01_build-from-source/index.md

```diff
@@ -2,123 +2,19 @@
 content_title: Build Antelope from Source
 ---
 
-The shell scripts previously recommended for building the software have been removed in favor of a build process entirely driven by CMake. Those wishing to build from source are now responsible for installing the necessary dependencies. The list of dependencies and the recommended build procedure are in the README.md file. Instructions are also included for efficiently running the tests.
+The shell scripts previously recommended for building the software have been removed in favor of a build process entirely driven by CMake. Those wishing to build from source are now responsible for installing the necessary dependencies. The list of dependencies and the recommended build procedure are in the [`README.md`](../../../README.md) file. Instructions are also included for efficiently running the tests.
```

Comment:
We must use absolute URLs to prevent the link from breaking on the docs portal:
https://github.com/AntelopeIO/leap/blob/release/3.1/README.md

Reply:
Done.